tests/integration/cmd_link_unlink_test.go

```diff
@@ -9,138 +13,199 @@ import (
 
 var _ = Describe("odo link and unlink command tests", func() {
 
-	/*
-		Uncomment when we uncomment the test specs
-		var frontendContext, backendContext string
-		var oc helper.OcRunner
-	*/
-
 	var commonVar helper.CommonVar
 
-	// This is run before every Spec (It)
 	var _ = BeforeEach(func() {
-		// oc = helper.NewOcRunner("oc")
 		commonVar = helper.CommonBeforeEach()
+		// wait till odo can see that all operators installed by setup script in the namespace
```

Comment:
It has timeout. It will not wait forever. Comment seems confusing to me. Could you please have a look